app.js

```diff
@@ -2,7 +2,10 @@
 
 require('dotenv').config()
 
-const fs = require('fs')
+const { KthAppinsights } = require('@kth/appinsights')
+
```

Comment:
```suggestion
```

Reply:
Just to be consistent? Or was this because of eslint?

Reply:
yes, because of eslint